assets/src/dashboard/components/cardGallery/index.js

```diff
@@ -143,7 +151,13 @@ function CardGallery({ story }) {
             height: metrics.activeCardSize.height,
           }}
         >
-          <ActiveCard {...metrics.activeCardSize}>
+          <ActiveCard
+            aria-label={
+              __('Active Page Preview - Page', 'web-stories') +
+              ` ${activePageIndex + 1}`
+            }
```

Comment:
I think something like this would be more in line with how this i18n package works. 
```
aria-label={sprintf(
                /* translators: %s: active preview page number */
                __('Active Page Preview - Page %s', 'web-stories''),
                activePageIndex + 1
              )}
```

Reply:
Yes please!

Never do string concatenation for translatable strings. Happy to provide some resources about this, but the gist is:

Imagine you need to translate something to another language.

Would you rather want to translate this complete sentence: `__( 'Your city is %1$s, and your zip code is %2$s.', 'web-stories' )` or these two concatenated strings:

`__( 'Your city is ', 'web-stories' ) + city + __( ', and your zip code is ', 'web-stories' ) + zipCode`

You might succeed to translate this into _some_ languages, but it would be a really bad experience because you don't have the full context of the sentence.